exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -157,6 +158,21 @@ func (ku *Kucoin) SetDefaults() {
 				GlobalResultLimit: 1500,
 			},
 		},
+		Subscriptions: []*subscription.Subscription{
```

Comment:
For some reason I am not getting a panic, this is not happening on master:

```
panic: runtime error: index out of range [1] with length 1

goroutine 278 [running]:
github.com/thrasher-corp/gocryptotrader/exchanges/kucoin.(*Kucoin).wsHandleData(0xc000004300, {0xc001020600, 0x1f, 0x200})
        thrasher-corp/gocryptotrader/exchanges/kucoin/kucoin_websocket.go:209 +0x14c5
github.com/thrasher-corp/gocryptotrader/exchanges/kucoin.(*Kucoin).wsReadData(0xc000004300)
        thrasher-corp/gocryptotrader/exchanges/kucoin/kucoin_websocket.go:188 +0x8f
created by github.com/thrasher-corp/gocryptotrader/exchanges/kucoin.(*Kucoin).WsConnect in goroutine 162
        thrasher-corp/gocryptotrader/exchanges/kucoin/kucoin_websocket.go:132 +0x352
```

Some print lines around that area of issue:

```
raw: {"id":"169610345","type":"ack"}
topicInfo: []
resp.Topic: ""
```

Reply:
Thanks. On it.

Reply:
resp.ID handling needed to be fixed, instead of removed.
The IncomingData matcher needed to be unconditional on a wait map.

Fixed eecea484